### PR TITLE
Create ROLib.netkan

### PR DESCRIPTION
**Pull request by DRVeyl**

### CODE REVIEW

File: NetKAN/ROLib.netkan [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "recommends" : [
+        { "name" : "ROEngines" },
+        { "name" : "ROTanks" },
+        { "name" : "ROSolar" },
+        { "name" : "ROCapsules" },
+        { "name" : "RealismOverhaul" },
+        { "name" : "RP-0" }
+    ],
+    "conflicts" : [
+        { "name" : "TweakableEverything" }
```
Comment (HebaruSan) [inline NetKAN/ROLib.netkan:30]:
```suggestion
        { "name" : "TweakableEverythingCont" }
```

`TweakableEverything` isn't listed, but `TweakableEverythingCont` is.

Comment (DRVeyl) [reply NetKAN/ROLib.netkan:30]: Thanks for catching that.  Will make sure that gets incorporated into later PRs. :)

**Comment by DRVeyl** [comment]
... And now I've remembered that I'm supposed to hit the button to accept the change!

**Comment by HebaruSan** [comment]
Looks good, thanks for the submission.

**Pull request merged**